external/DoCIF

```diff
@@ -1 +1 @@
-Subproject commit e52def19124bec4c9b743a8f7276ef2b14f64b51
+Subproject commit 1dec20b1995d4735b7ec295a6ec4ae4d584ee187
```

Comment:
I wasn't sure, It caused a conflict and I thought I used the current one off master. Either way, I'll change it back.